sdk/servicebus/service-bus/test/atomManagement.spec.ts

```diff
@@ -672,7 +520,9 @@ const alwaysBeExistingRule = "alwaysbeexistingrule";
         "_response",
         "createdOn",
         "updatedOn",
-        "accessedOn"
+        "accessedOn",
+        "forwardTo",
+        "forwardDeadLetteredMessagesTo"
```

Comment:
In general, these should be excluded from being compared, since just like other properties such as timestamps, there;s no point comparing since the values would be different (And see other comment, it's practically not possible to use absolute URLs unless maybe there's some dedicated identifier we could use representing the endpoint). Thoughts?

Reply:
> there;s no point comparing since the values would be different 

The concern was around excluding properties that arent meant to exist in the response in the first place. 

This section is for testing the `createTopic()` method. From what we know of the `forward` related properties they are settable and readable only from queues and subscriptions.

Does the service return these properties in the response when creating topic too?

> it's practically not possible to use absolute URLs unless maybe there's some dedicated identifier we could use representing the endpoint

The endpoint is available from the connection string being used in the tests isnt it?

Reply:
Removing to unblock, it's largely just unnecessary and my intention was that it could have helped with if we refactor in future to hold a list of all properties that don't have 1:! mapping between inputs and output.

> The endpoint is available from the connection string being used in the tests isnt it?

Sure, the endpoint can be extracted, needs to use the parseConnectionString utility - will update to add in tests for the absolute URL cases